itests/common.bash

```diff
@@ -248,6 +248,7 @@ teardown() {
 
   log "Running docker-compose down"
   TEST_NAME=$BATS_TEST_DESCRIPTION docker-compose down
+  sleep 5
```

Comment:
I don't love this because it will add a cumulative 70 seconds to the existing build. In addition `START_SLEEP` will add another 70.

Reply:
They weren't strictly necessary and were there for isolation before. Removed.